src/network/xmpp/handlers/historyMessageHandler.ts

```diff
@@ -145,3 +151,14 @@ export function onRequestSingleMessage(stanza: Element, messageWithResponseId: s
 	const store: RootStore = useStore.getState();
 	store.setRepliedMessage(referenceMessage.roomId, messageWithResponseId, referenceMessage);
 }
+
+export function onLoadFullHistory(stanza: Element): void {
+	xmppDebug('Request full history', stanza);
+	const roomId = getId(getRequiredAttribute(stanza, 'from'));
+	const { chatExporting } = useStore.getState().session;
+
+	if (chatExporting && chatExporting.roomId === roomId) {
```

Comment:
you can simplify with if (chatExporting?.roomId === roomId)

if (chatExporting?.roomId === roomId) {
		const isHistoryComplete = !!getRequiredTagElement(stanza, 'fin').getAttribute('complete');
		chatExporting.exporter.handleFullHistoryResponse(isHistoryComplete);
}